### DISCUSSION TITLE
<Field>: Put the helperText on a new line if the orientation is horizontal.

### DISCUSSION THREAD

**Discussion by du33169** [chakra-ui/chakra-ui#9344] [General]
### Description

If I set the orientation of <Field> to horizontal, I expect only the label and content to be horizontally aligned. The helperText should always be on a new line for a better visual appearance. Currently the HelperText is also squeezed into the same line, which does not look very elegant.

### Link to Reproduction

https://stackblitz.com/edit/chakra-ui-v3-cxtanb3x?file=src%2FApp.tsx

### Steps to reproduce

As described above.

### Chakra UI Version

3.2.2

### Browser

Mozilla Firefox v133.0

### Operating System

- [ ] macOS
- [X] Windows
- [ ] Linux

### Additional Information

currently:
![scr](https://github.com/user-attachments/assets/fe41c9c0-7829-4dc9-a06a-54d942016cc6)

expected :

![scr](https://github.com/user-attachments/assets/99574168-12c5-4e2a-82bb-bda3d7e73f12)

**Comment by segunadebayo** [comment]
Feel free to tweak the Field snippet to match your needs. The code is yours.

Alternatively, you can import the Field component directly from chakra and compose as you wish

**Comment by jorge-romo** [comment]
Same issue here. I was able to modify the styles of certain components within Field (e.g., Field.Label, Field.HelperText, Field.ErrorText), but not the Input component since it's deeply nested within children (it could be a child, grandchild, or even deeper). The challenge is that we don't have direct access to that Input.
I also tried modifying the recipe since Field has an input slot, but it didn’t have any effect.

https://stackblitz.com/edit/chakra-ui-v3-1igyaggf?file=src%2Fcomponents%2Fui%2Ffield.tsx,src%2FApp.tsx

**Reply by jorge-romo** [reply]
I figured out the solution by using render function prop instead of using children.
Solution: https://stackblitz.com/edit/chakra-ui-v3-jr2wqmxw?file=src%2Fcomponents%2Fui%2Ffield.tsx,src%2FApp.tsx

**Comment by segunadebayo** [comment]
The Field snippet is yours. Change the layout there.

Or compose `Field` parts yourself and skip the snippet.
